server/src/index.js

```diff
@@ -35,20 +43,17 @@ async function start() {
   app.get('/health', (req, res) => res.sendStatus(200))
 
   // connect to db
-  const dataSource = {
-    client: await connect(config.db),
-    type: 'knex'
-  }
+  const client = await connect(config.db);
 
+  const schema = appModule.schema;
   const apolloConfig = {
-    typeDefs,
-    resolvers,
+    schema,
```

Comment:
It feels like passing schema is not supported. Dropping GraphQL-modules for the moment

@StephenCoady Did we made that change to pass schema ?

Reply:
@wtrocki no, we reverted it to be just types and resolvers. we refactored voyager-tools so that subscriptions in resolvers were handled differently.

Reply:
Ok. @darahayes I think in long run it will be nice to have both. If you agree I could create bug.